novawallet/Modules/Staking/Dashboard/Model/StakingDashboardModel.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import RobinHood
+
+struct StakingDashboardItemModel {
+    let stakingOption: Multistaking.ChainAssetOption
+    let dashboardItem: Multistaking.DashboardItem?
+    let accountId: AccountId?
+    let balance: AssetBalance?
+    let price: PriceData?
+    let isOnchainSync: Bool
+    let isOffchainSync: Bool
+
+    var hasStaking: Bool {
+        guard let dashboardItem = dashboardItem else {
+            return false
+        }
+
+        return dashboardItem.stake != nil
+    }
+
+    func balanceValue() -> Decimal {
```

Comment:
looks like a property

Reply:
I think we should mark as property the logic that for constant time, otherwise better to use functions